src/models/user.ts

```diff
@@ -72,10 +73,16 @@ const UserSchema = new mongoose.Schema(
   },
 );
 
-UserSchema.virtual('event', {
-  ref: 'Event',
```

Comment:
This was from the `mern-auth` template. Can be removed no prob

Reply:
Big up @`mern-auth`